alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
+                --padding: 6px;
+            }
+            
+            .container {
```

Comment:
When using Shady-DOM (Polymer's light-weight polyfill for Shadow-DOM), these styles are "scoped" and added to the header.  These styles are in effect leaky, and because Polymer scopes them, the risk of affecting other elements is minimized.  On the other hand, the "shadow-DOM" (or what would normally be shadow-DOM) is exposed as light-DOM and subject to be affected by other styles on the page.  For instance, if some other style on the page defined a `.container` CSS selector, it would affect this WC DOM rendering.  For this reason, I would recommend choosing class names that will be safe in that rendering context.